ProcessQueue/__init__.py

```diff
@@ -476,3 +479,71 @@ def main(msg: func.QueueMessage):
             field_ids = []
             field_names = []
             data = []
+    if parent_sr_id is not "None":
+        # Grab Parent Scan Report
+        parent_scan_report = requests.get(
+                    url=api_url
+                    + "scanreporttablesfilter/?scan_report="
+                    + str(parent_sr_id),
+                    headers=headers,
+                )
+        parent_table_ids=[]
+        parent_tables=[]
+        child_tables=[]
+        
+        child_scan_report=requests.get(
+                    url=api_url
+                    + "scanreporttablesfilter/?scan_report="
+                    + str(scan_report_id),
+                    headers=headers,
+                )
+        
+        response_parent = json.loads(parent_scan_report.content.decode("utf-8"))
+        response_child=json.loads(child_scan_report.content.decode("utf-8"))
+
+        print("PARENT SCAN REPORT TABLES:",response_parent)
+        print("CHILD SCAN REPORT TABLES",response_child)
+
+        # Find differences between JSON?
+        # difference=diff(response_parent, response_child)
+        # print("JSON DIFF",difference)
+
+        # Find differences of two lists?
+        for element in range(len(response_parent)):

```

Comment:
pedantic comment ...

can you just not iterate over the element instead of the index?
i.e.
```
for element in response_parent:
      parent_tables.append(element["name"])
```
or
parent_tables = [
element['name']
for element in response_parent
]

Reply:
@calmacx yeah I was looking for a better solution for that but I iterate over the index because response_parent is a JSON dict and you can't pass a dictionary as a list indice

Reply:
ah yeh, you probably want
```
for key,element in response_parent.items()
```
or 
```
for element in response_parent.values()
```
when iterating over a json/dict